README.md

```diff
@@ -65,6 +65,31 @@ Currently the waiting period is as follows:
 
 ## FAQ
 
+#### Do I need to register a package to install it?
+
+No, you can simply do `using Pkg; Pkg.add(url="https://github.com/JuliaLang/Example.jl")`
+to e.g. install the package `Example.jl`, even if it was not registered.
+
+#### Should I register my package?
+
+If you think your package might be useful to others, or provide functionality other
+registered packages might want to rely on, go for it!
+
+It is easier for others to use your package if it has documentation that explains
```

Comment:
Maybe the bit below this this should be it's own question.
"What should I do to before registering a package?"

Maybe it should just be dropped as beyond the scope of this document?

Reply:
hm, yeah. I meant this as kind of an elaboration on what does it mean to be useful to others. Like here's some ways you can help make it useful (and to encourage best practices).

I don't totally think it's beyond the scope of the document, because we do want packages in General to do these things. E.g. there's issues on RegistryCI for requiring X lines of tests (which IMO could also be "Y lines of README or docs"). So if it's in-scope to be a hard requirement, I think an earlier step is for it to be a soft recommendation and it should be in-scope for that.

Reply:
I tried to reword it a bit so it is hopefully more encouraging and also more direct